apply/views.py

```diff
@@ -1,3 +1,79 @@
-from django.shortcuts import render
+import json
 
-# Create your views here.
+from django.http        import JsonResponse
+from django.views       import View
+from django.db.models   import Count
+
+from .models        import (
+        Apply,
+        ProcessStatus
+)
+from user.models    import User
+from utils          import login_decorator
+from resume.models  import Resume
+from posting.models import (
+        Posting,
+        Occupation,
+        JobCategory,
+        CompanyImage,
+        CompanyDetail,
+        State,
+        County,
+        Tag,
+        TagDetail,
+        CompanyTag,
+        BookMark,
+        Like,
+        Company
+        )
+
+class MyWantUsView(View):
+    @login_decorator
+    def get(self, request):
+        user    = request.user
+        user_info = {
+                'profile'       : user.image_url,
+                'name'          : user.name,
+                'email'         : user.email,
+                'phoneNumber'   : user.phone_number
+                }
+    
+        apply_list = {
+                "stepOne"   : Apply.objects.filter(user=user).filter(process_status_id=1).count(),
+                "stepTwo"   : Apply.objects.filter(user=user).filter(process_status_id=2).count(),
+                "stepThree" : Apply.objects.filter(user=user).filter(process_status_id=3).count(),
+                "stepFour"  : Apply.objects.filter(user=user).filter(process_status_id=4).count()
+                }
```

Comment:
```suggestion
        apply_list = {
                "stepOne"   : Apply.objects.filter(user=user, process_status_id=1).count(),
               
                }
```

1. filter 사용할 때 filter를 계속 안 붙이고 filter 안에 (,)로 사용할 수 있습니다.
2. Apply 객체로 말고, User 객체를 시작으로 count() 해보세요.

Reply:
!!!! 감사합니다 !!!! 수정했어요!!!